Dockerfile

```diff
@@ -14,48 +11,50 @@ RUN make install
 
 # ##################################
 
-FROM debian:buster as hub
+FROM debian:buster as fetchd
+
+ARG DEFAULT_RPC_ENDPOINT=""
+ARG DEFAULT_SEEDS=""
 
-# Set up dependencies
-ENV PACKAGES jq curl
+ENV RPC_ENDPOINT="${DEFAULT_RPC_ENDPOINT}"
+ENV SEEDS="${DEFAULT_SEEDS}"
 
 RUN apt-get update && \
-    apt-get install -y $PACKAGES
+    apt-get install -y jq curl
 
 COPY --from=builder /go/pkg/mod/github.com/\!cosm\!wasm/go-cosmwasm@v*/api/libgo_cosmwasm.so /usr/lib/libgo_cosmwasm.so
 COPY --from=builder /go/bin/fetchcli /usr/bin/fetchcli
 COPY --from=builder /go/bin/fetchd /usr/bin/fetchd
 COPY entrypoints/entrypoint.sh /usr/bin/entrypoint.sh
 
 VOLUME /root/.fetchd
-VOLUME /root/secret-temp-config
```

Comment:
Can you double check that we no longer use this path in our developments @daeMOn63 . I seem to remember that it was added for this reason. Although I am very much in favour of removing this

Reply:
it's only used on internal builds. The public version doesn't need that. I've restored those volumes definition in their matching target, but imo this is deployment-related data, not something the application needs to know about.